app/src/components/copy-button.tsx

```
@@ -0,0 +1,58 @@
+    useEffect(() => {
+        if (isCopied) {
+            const timer = setTimeout(() => {
+                setIsCopied(false);
+            }, 2000);
+
+            return () => {
+                clearTimeout(timer);
+            };
+        }
+
+        return () => {};
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)